core/src/main/scala/annotations.scala

```diff
@@ -40,17 +50,26 @@ package object annotation {
   object commandMacro {
     def impl(c: Context)(annottees: c.Expr[Any]*): c.Expr[Any] = {
       import c.universe._
+
       val result = annottees.map(_.tree).toList match {
         case q"$annots def $methodName(...$args): $tpe = $body" :: nil =>
           val allArgs = args.map{
             case innerArgs => q"val actionCommand: String" +: innerArgs
           }
-          q"$annots def $methodName(...$allArgs): $tpe = $body"
+          val operationName = c.universe.TermName(chooseOperationName(
+            c = c,
+            methodName = methodName.toString
+          ))
+          q"$annots def $operationName(...$allArgs): $tpe = $body"
         case q"$annots def $methodName(...$args): $tpe" :: nil =>
           val allArgs = args.map{
             case innerArgs => q"val actionCommand: String" +: innerArgs
           }
```

Comment:
I've already been thinking about how to improve the readability of this module, and it's kind of tricky.

- Quasiquotes are pretty dense, I'm not sure introducing a method would make code more readable (the signature would be awful)
- It not that easy to generalize the method for both `command` and `query` (the use different `c.universe`)